xAPI.md

```diff
@@ -2690,6 +2723,12 @@ application needs while minimizing the potential for misuse. The limitations of
 are in addition to any security limitations placed on the user account associated with the
 request.
 
+Elements of this section draw on [OAuth 2.0](http://tools.ietf.org/html/rfc6749#section-3.3)
+despite this section describing requirements for LRS supporting [OAuth 1.0](http://tools.ietf.org/html/rfc5849). 
```

Comment:
This is an attempt to avid the confusion when people see OAuth 2.0 referenced in a section about OAuth 1.0. Suggestions for alternate wording welcome!